ain/db/ref.py

```diff
@@ -413,6 +414,48 @@ async def matchOwner(self, params: MatchInput = None) -> Any:
             ref = Reference.extendPath(ref, getattr(params, "ref", None))
         return await self._ain.provider.send("ain_matchOwner", {"ref": ref})
 
+    async def getStateProof(self, params: StateInfoInput = None) -> Any:
+        """Fetches the state proof of a global blockchain state path.
+
+        Args:
+            params (StateProofInput): The state info input object.
```

Comment:
it is different from the actual parameter type.

Reply:
Just fixed. Thanks!